wgpu-native/Cargo.toml

```diff
@@ -17,7 +17,7 @@ crate-type = ["lib", "cdylib", "staticlib"]
 
 [features]
 default = []
```

Comment:
What's the reason to move to `local` instead of `remote`? I could see usefulness in being able to opt out of remote-specific concepts if we expose more in wgpu-native (even though we don't have much currently). Either way it might still be good to list `local` as a default feature to make it easier for people to build wgpu-native directly (thinking of https://nest.pijul.com/porky11/wgpu for example)

Reply:
I think that's a good idea!

Reply:
The answer is - remote used to add some extra capabilities, like the Serde derives. As I was trying to integrate it into Gecko, I realized that using Gecko's IPC is the only way. That means Serde is no longer useful, or at least, doesn't get associated with "remote" feature.

At the same time, this PR gets rid of the static GLOBAL in general case. So it's "local" that adds features now: it exposes C entry points and declares a lazy static used in them.